.github/workflows/api-tests.yml

```diff
@@ -108,7 +108,7 @@ jobs:
 
       - name: Poetry check
         run: |
-          poetry check -C api
+          poetry check -C api --lock
```

Comment:
declaimer: the `--lock` option does not change the lockfile , and it only makes sure the lockfile exists.